lib/Dist/Zilla/Plugin/Test/Perl/Critic.pm

```diff
@@ -75,14 +75,25 @@ sub register_prereqs {
 }
 
 sub _dumper {
-    my ($value) = @_;
+    my ($value, $bare) = @_;
     local $Data::Dumper::Indent = 1;
     local $Data::Dumper::Useqq = 1;
     local $Data::Dumper::Terse = 1;
     local $Data::Dumper::Sortkeys = 1;
     local $Data::Dumper::Trailingcomma = 1;
     my $dump = Data::Dumper::Dumper($value);
     $dump =~ s{\n\z}{};
+    if ($bare) {
```

Comment:
What's actually going on in here? This is getting complicated enough that it needs tests. Parsing Data::Dumper output seems like the wrong place to be munging data.

Reply:
It's producing a list rather than an arrayref or a hashref. The alternative is producing code that looks like `@{[ ... ]}` and `%{{ ... }}`.

Would you prefer that I remove it?

I can add tests otherwise.

Reply:
The minimum perl prereq can be raised to 5.20, so that allows for `->@*` constructs everywhere, which likely would be even easier still.

Is the `$bare` argument needed? When would it not be passed? The template calls it in two places, both with this value set; would it ever be useful for a user to provide a template where this isn't set?

Reply:
The ref+deref code is in the generated test. The generated test doesn't have a perl 5.20 prereq.

It felt weird to have a sub named `_dumper` that didn't produce roughly "Data::Dumper" output, so I added an option as hopefully a clue that it's doing something special.

Reply:
I've just removed this and used a deref.